### PR TITLE
some refactoring

### PR DESCRIPTION
**Pull request by itsabdessalam**
🚨 `theme` field doesn't seem to be returned by the API anymore (@codebynao)

### CODE REVIEW

File: src/components/NoteForm.jsx [hunk 1]
```diff
@@ -30,59 +28,78 @@ const StyledNoteForm = styled.div`
...
-        {error ? <p className="error">{error.message}</p> : null}
-        <Button type="submit" disabled={isLoading}>
-          {isLoading ? "Loading" : "Create"}
+        <TemplateSelector
+          onChange={handleOnChange}
+          template={formData.template}
+        />
+        {status.error ? <p className="error">{status.error}</p> : null}
+        <Button type="submit" disabled={status.loading}>
+          {status.loading ? "Loading" : "Create"}
```
Comment (codebynao) [inline src/components/NoteForm.jsx]: Shouldn't it be translated?

**Review by codebynao — COMMENTED** [review]
Just one question about translation
Otherwise everything looks great!

> 🚨 theme field doesn't seem to be returned by the API anymore (@codebynao)

The theme is not saved in DB, last time I asked, we decided to not save this information and only cache it

**Comment by codebynao** [comment]
Anyway I updated the user so now we're saving the theme in DB haha
It would be less confusing this way

File: src/components/NoteForm.jsx (hunk 1 above)
Comment (itsabdessalam) [reply src/components/NoteForm.jsx]: Ooops 🤭
I'll fix it

**Comment by itsabdessalam** [comment]
> Anyway I updated the user so now we're saving the theme in DB haha
> It would be less confusing this way

I didn't know 😔
Thanks!

File: src/components/NoteForm.jsx (hunk 1 above)
Comment (itsabdessalam) [reply src/components/NoteForm.jsx]: Fixed

**Review by codebynao — APPROVED** [review]
Thanks! 😁

**Pull request merged**